contracts/mocks/ERC165Mock.sol

```diff
@@ -2,9 +2,9 @@
 
 pragma solidity ^0.8.0;
 
-import "../introspection/ERC165.sol";
+import "../introspection/ERC165Storage.sol";
 
-contract ERC165Mock is ERC165 {
+contract ERC165Mock is ERC165Storage {
```

Comment:
This should now be `ERC165StorageMock`, and the test should be `ERC165Storage.test.js`, with `contract('ERC165Storage', ...`.